sys-apps/plocate/files/plocate-1.1.16-include-linux-stat_h-musl.patch

```diff
@@ -0,0 +1,26 @@
+From 0125004cd28c5f9124632b594e51dde73af1691c Mon Sep 17 00:00:00 2001
```

Comment:
No links?

Reply:
whops, dropped them when I wgetted from his git (which replaced my patch).

Reply:
@thesamesam: now it's good I think